src/middleware/records.middleware.js

```diff
@@ -0,0 +1,33 @@
+export const validateGroupId = (req, res, next) => {
+    const { groupId } = req.params;
+    if (isNaN(parseInt(groupId, 10))) {
+        return res.status(400).json({ message: 'URL 경로 그룹 ID가 유효하지 않습니다.' })
+    }
+    
+    next();
+};
+
+export const validateRecordBody = (req, res, next) => {
+    const { authorNickname, authorPassword, exercise_type, time, distance } = req.body;
```

Comment:
```suggestion
    const { authorNickname, authorPassword, exerciseType, time, distance } = req.body;
```
exercise_type으로 받으시면 undefinde가 들어올 것 같습니다!